test/gpu/jit.cpp

```diff
@@ -206,10 +206,11 @@ TEST_CASE(compile_warnings)
     EXPECT(not compile("").empty());
     EXPECT(not compile("-Wunused-parameter -Wno-error").empty());
     EXPECT(not compile("-Wno-unused-parameter -Werror").empty());
-#ifndef MIGRAPHX_USE_HIPRTC
-    EXPECT(test::throws([&] { compile("-Werror=unused-parameter"); }));
-    EXPECT(test::throws([&] { compile("-Wunused-parameter -Werror"); }));
-#endif
+    if(migraphx::disabled(migraphx::gpu::MIGRAPHX_ENABLE_HIPRTC_WORKAROUNDS{}))
```

Comment:
It should be `if (not migraphx::enabled(migraphx::gpu::MIGRAPHX_ENABLE_HIPRTC_WORKAROUNDS{}))`. Using `disabled` would require setting the env variable to enable the tests(ie `MIGRAPHX_ENABLE_HIPRTC_WORKAROUNDS=0`). We want the tests enabled by default without any env variables and only skipped when `MIGRAPHX_ENABLE_HIPRTC_WORKAROUNDS=1` is set.

Reply:
Done.